opencxl/cxl/device/cxl_type1_device.py

```diff
@@ -0,0 +1,125 @@
+"""
+ Copyright (c) 2024, Eeum, Inc.
+
+ This software is licensed under the terms of the Revised BSD License.
+ See LICENSE for details.
+"""
+
+# pylint: disable=duplicate-code, unused-import
+from asyncio import create_task, gather
+from dataclasses import dataclass
+from typing import Optional
+
+from opencxl.util.component import RunnableComponent
+from opencxl.cxl.component.cxl_connection import CxlConnection
+from opencxl.cxl.component.cxl_cache_manager import CxlCacheManager
+from opencxl.cxl.component.cxl_io_manager import CxlIoManager
+from opencxl.cxl.mmio import CombinedMmioRegister, CombinedMmioRegiterOptions
+from opencxl.pci.component.pci import (
+    PciComponent,
+    PciComponentIdentity,
+    EEUM_VID,
+    SW_SLD_DID,
+    PCI_CLASS,
+    MEMORY_CONTROLLER_SUBCLASS,
+)
+from opencxl.pci.component.mmio_manager import MmioManager, BarEntry
+from opencxl.pci.component.config_space_manager import (
+    ConfigSpaceManager,
+    PCI_DEVICE_TYPE,
+)
+
+
+@dataclass
+class SingleLogicalDeviceConfig:
+    serial_number: str
+    port_index: int
+    memory_size: int  # in bytes
+    memory_file: str
+    vendor_id: int = EEUM_VID
+    device_id: int = SW_SLD_DID
+    subsystem_vendor_id: int = 0
+    subsystem_id: int = 0
```

Comment:
Remove. SLD is a Type 3 device. Type 1 devices do not have its own memory.

Reply:
Removed.